src/lmsr.js

```diff
@@ -19,7 +19,7 @@ export function calcLMSRCost (opts) {
         netOutcomeTokensSold.reduce((acc, numShares, i) =>
             acc.plus(
                 new Decimal(numShares.toString())
-                .plus(i === outcomeTokenIndex ? outcomeTokenCount : 0)
+                .plus(i === parseInt(outcomeTokenIndex, 10) ? outcomeTokenCount : 0)
```

Comment:
Yes, it would, but it would also match a whole lot of other things that would be wrong:
```
> "" == 0
true
> [] == 0
true
> false == 0
true
> true == 1
true
```

I think it's good to enforce triple equality checks here because loose comparison in JS is kinda... quirky:
https://dorey.github.io/JavaScript-Equality-Table/